packages/core/admin/admin/src/pages/ProfilePage/tests/__snapshots__/index.test.js.snap

```diff
@@ -1,6 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`ADMIN | Pages | Profile page renders and matches the snapshot 1`] = `
+exports[`ADMIN | Pages | Profile page | without SSO lock renders and matches the snapshot 1`] = `
```

Comment:
Same question again: do you think this snapshot test is useful? I'd remove it personally, because it's huge and I don't see why it helps us.

Reply:
I removed the snapshot test